src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/crumbTable.jsx

```diff
@@ -29,11 +31,19 @@ class CrumbTable extends React.Component {
   };
 
   render() {
+    const platform = this.context;
+    const widerCategory = platform === 'csharp';
+
     return (
       <table className="table key-value">
         <thead>
           <tr>
-            <td className="key">
+            <td
+              className="key"
+              css={
+                widerCategory && {maxWidth: '215px !important', width: '215px !important'}
```

Comment:
Does 215 need to change in smaller viewports?

Reply:
thanks, changed this to styled component with media query 👍